### ISSUE TITLE
SyntaxError when running aemservice task

### ISSUE THREAD

**Issue by sebastianwr** [pmarkham/ansible-adobe-aem#1]
Hi Paul,

first of all, thank you very much for the work on this project. Looks absolutely amazing and will be a great help in the future.

I just started to use this and encountered the following error when running a basic playbook to restart a server:

```
failed: [192.168.89.14] => {"failed": true, "parsed": false}
BECOME-SUCCESS-jqwgnqznhbfwqraipjjdgyyxwhhqvuax
  File "/home/swramba/.ansible/tmp/ansible-tmp-1446638597.59-167390120721752/aemservice", line 231
    except Exception as e:
                      ^
SyntaxError: invalid syntax
OpenSSH_6.9p1, LibreSSL 2.1.7
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 21: Applying options for *
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 192.168.89.14 closed.


FATAL: all hosts have already failed -- aborting
```

Is there something wrong on the node? Have you encountered this before?
I also had this issue with the older ansible-adobe-cq project but not with the yum command for example.

Seems to be an issue with the python installed on the node. Here is some system info:

```
# yum list installed | grep python
audit-libs-python.x86_64                   1.8-2.el5                   installed
dbus-python.x86_64                         0.70-9.el5_4                installed
libselinux-python.x86_64                   1.33.4-5.7.el5              installed
libxml2-python.x86_64                      2.6.26-2.1.25.el5_11        installed
python.x86_64                              2.4.3-56.el5                installed
python-dmidecode.x86_64                    3.10.13-1.el5_5.1           installed
python-elementtree.x86_64                  1.2.6-5                     installed
python-iniparse.noarch                     0.2.3-6.el5                 installed
python-libs.x86_64                         2.4.3-56.el5                installed
python-simplejson.x86_64                   2.0.9-8.el5                 installed
python-sqlite.x86_64                       1.1.7-1.2.1                 installed
python-urlgrabber.noarch                   3.1.0-6.el5                 installed
rpm-python.x86_64                          4.4.2.3-36.el5_11           installed
```

```
# cat /proc/version 
Linux version 2.6.18-404.el5 (mockbuild@x86-033.build.eng.bos.redhat.com) (gcc version 4.1.2 20080704 (Red Hat 4.1.2-55)) #1 SMP Sat Mar 7 04:14:13 EST 2015
```

**Comment by sebastianwr** [comment]
Solved it by installing Python 2.6. The "as" is a Python 3.x feature which has been backported only up to 2.6.

**Comment by pmarkham** [comment]
Thanks for the feedback. I wasn't aware this was a Python 2.6 only feature. I'll look at updating it to work with Python 2.4 when I get a chance.
